### DISCUSSION TITLE
Boiler compatibility list

### DISCUSSION THREAD

**Discussion by sergeantd83** [Alexwijn/SAT#21] [General]
This is a list of boiler brand / model that provenly works with SAT.
If your boiler works with SAT please comment below your Boiler’s Brand and Model.

Gas Boilers
| Brand - Model | ID3:LB | 
| --- | :---: |
| Baxi Luna Dua-Tec | |
| Dedietrich MCR3 Evo | 11* |
| Ferroli BlueHelix Pro | 9 |
| Ferroli BlueHelix Tech RRT | 9 |
| Geminox | 4 | 
| Ideal Logic ESP1 | 6 |
| Ideal Logic Heat | 6 |
| Ideal Vogue Max | 6 |
| Immergas Victrix Omnia | 27** |
| Immergas Victrix Superior | 27** |
| Intergas HRE | 173 |
| Intergas Xtreme | 173 |
| Itho Daalderop Base Cube | 29 |
| Nefit Pro-Line HRC | 131 |
| Radiant R2K | 41 |
| Remeha Avanta CW5 | 11* |
| Remeha Calenta | 11* |
| Sime Vera HE | 27** |
| Saunier Duval ThemaFast Condens 25 using the AWB Ebus Opentherm interface 0020092426 | - |
| Termet Ecocondens Gold Plus | - |
| Vaillant VHR Solide Plus | 24 |
| Viessmann Vitodens 100-W | 33 |

Heat Pumps
| Brand - Model | ID3:LB | 
| --- | :---: |
| Remeha Elga Ace | 11* |

*Dedietrich and Remeha have the same MemberID Code
**Immergas and Sime have the same MemberID Code
***This list will be updated

**Comment by pbajdor** [comment]
Works perfect with Viessmna Vitodens 100-W.

**Reply by slavo-83** [reply]
Ferroli BlueHelix PRO

**Reply by sergeantd83** [reply]
@pbajdor can you please provide your `slave memberID code` ( ID3:LB )?

**Comment by mezger6** [comment]
Working great with a Radiant R2K24

**Reply by sergeantd83** [reply]
@mezger6 can you please provide your slave memberID code ( ID3:LB )?

**Reply by mezger6** [reply]
Sure, could you give me a hint on where to find it?

**Reply by sergeantd83** [reply]
It depends on which OTGW version you have.

WiFi version
`sensor.otgw_status_slave_memberid_code`

Serial version `sensor.slave_memberid_boiler_xxxx`

**Reply by mezger6** [reply]
I don't have either entity. My setup is OTGW board via serial to server with otmonitor with message relay, its socket configured as OTGW serial in SAT. No ESP in between, but I plan to incorporate one some time in the future - if it's necessary to get this info I'll get back to you then.

**Reply by sergeantd83** [reply]
Ok, then you should send the PM=3 command through OTmonitor. Then look at the statistics tab. Look for the ID3. ID3 is displayed as `HB LB`, so it should look like this `XXXXXXXX XX`. Hope this helps 🙂

**Comment by mezger6** [comment]
This is what I'm getting in response to PM=3, hope this is what you need:
10:42:07.933342	BC0030129	Read-Ack  	Slave configuration (MsgID=3): 00000001 41
![image](https://github.com/Alexwijn/SAT/assets/8977619/f4d7afac-da9e-4b27-bfb8-dd10beab0c38)

**Reply by sergeantd83** [reply]
Yes, this is exactly what we want! Thank you very much!

**Reply by pbajdor** [reply]
03 	3 	Read 	Slave configuration 	7 	01010000 33

I hope this will help!

**Reply by sergeantd83** [reply]
Thank you very much both for your help!

**Comment by Richy22B** [comment]
If Intergas HRE and Xtreme work with this, so will the Intergas ECO RF, Xclusive & Rapid as they share the same OpenTherm programming.

**Reply by sergeantd83** [reply]
Correct, in theory SAT supports every  OpenTherm capable boiler / heat pump, but in this thread we record the boilers that already working with SAT.

**Reply by Richy22B** [reply]
I own https://theintergasshop.co.uk so will try it on our demo Intergas Xclusive when I get 5 minutes. My mum has an ECO RF and my mother-in-law has a Rapid 25, so I can try it on them and report back if you want 100% conformation.

**Reply by sergeantd83** [reply]
That will be a great confirmation, thanks! Intergas has a great innovation regarding instantaneous DHW production. Removing completely the 3-way valve and the heat exchanger, Intergas diminished the boiler maintenance. But there is only a small issue regarding the Intergas OpenTherm implementation, some boilers do not follow the zero modulation during low load control ( MM=0 ). As far as I know the Xtreme boilers after a software update changed this behavior and the zero modulation command is accepted by the boiler.

**Reply by Richy22B** [reply]
Please explain what you mean by "zero modulation" during low load control? As a side note, the Intergas Comfort Touch is a very interesting thermostat if you monitor its behaviour using an OpenTherm Gateway and the IDS software. One of the most advanced OpenTherm heating thermostats I have seen in operation in my opinion, although its features are limited. I also have an Intergas Xtend going into our training room shortly.

**Reply by sergeantd83** [reply]
There are 2 methods to control a boiler during low-load control.

1. The “official” OpenTherm method. That means the boiler not only accepts the change of `ID14` to zero but respects it also. That means when you set the ID14 to 0%, the maximum relative modulation stays always at 0% even if the CS is high ( E.g CS = 60 )
2. The workaround method. That means the boiler only accepts the change of `ID14` to 0 but the boiler doesn’t respect it. The workaround for this method is to quickly raise the CS at a value equal or lower by 1 K of the boiler flow water temperature. This workaround suppresses the modulation to 0%.

> One of the most advanced OpenTherm heating thermostats I have seen in operation in my opinion, although its features are limited.

Yes, Comfort touch algorithm is pretty good, I’ve seen some performance graphs during operation.

PS. We are working on a new feature that we will be able to identify which method every boiler follows. Then we will able to apply our strategy based on the results of this mechanism.

**Reply by Richy22B** [reply]
I'm familiar with the maximum relative modulation level (ID = 14) and I believe Intergas scrapped this somewhere around 2019. You used to be able to enable and disable this on the HRE and ECO RF boiler by changing boiler parameter Big C = 2, as I have used it with Honeywell thermostats which have low load control (evohome and T6). The burner would stay at minimum during low load control.

**Reply by sergeantd83** [reply]
Thanks for pointing this out. After a small research I found out that the C parameter accept only the values 0 and 1.
![IMG_3898](https://github.com/Alexwijn/SAT/assets/67208412/a7837afd-e9c1-4fd8-b5bf-5bfedfeb86c9)

**Reply by jophut** [reply]
I can confirm that my Intergas Kombi Kompakt HRE 36/30 A, production year 2019, with software release V2.53 (according to IDS, version 1.8) only accepts values 0 and 1 for parameter (Big) C.
Parameter (Big) C can NOT be set to value 2, neither via the button interface or IDS.

**Reply by sergeantd83** [reply]
Till now the “official” low-load control method is not working for immergas, intergas and ideal boilers. Thankfully @enryIT found a rather smart workaround to bring the “official” low-load control to immergas boilers. We do have some ideas about how to control boilers without ID14. I believe soon this issue will be resolved. It will be huge improvement for SAT since we will be able to support every OpenTherm boiler.

Thank you very much for the confirmation and the additional informations you provided.

**Reply by sergeantd83** [reply]
Moreover I would like to share an Intergas Xtreme behavior after a software update back in 2023 ( 18/4/2023 ). You can clearly see that the boiler respects the ID14.
![Intergas_Xtreme](https://github.com/Alexwijn/SAT/assets/67208412/a410d7e1-7c17-43e8-96ed-ab01c6303d04)

We also captured the software update moment.
![Interagas_Xtreme_software_update](https://github.com/Alexwijn/SAT/assets/67208412/f593bcdc-43c1-453c-9d76-3e56791e0ec3)

**Reply by Richy22B** [reply]
This is a snip from my own training manual from 2017. The Big C parameter was never documented properly and Intergas UK at the time had no idea of this functionality (nor much understanding about OpenTherm full stop). I did extensive research at the time on Intergas boilers for our own business and stumbled across Big C and its effect on low load control by complete accident. You can't access Big C = 2 from IDS, it can only be accessed from the HRE/ECO RF boiler display under installer menu 15. Like I said, I found it by complete luck and it was a shame it was removed in later models.
![snip-training-manual](https://github.com/Alexwijn/SAT/assets/77229024/134667ef-6c05-4c22-b7ed-fade2e00272c)

**Reply by Richy22B** [reply]
This is also a video I did from 6 years ago showing OTC control on a T4M thermostat (https://youtu.be/ZvjtkDWpjC8?si=hh70WILa7QaN3WlQ). All interesting stuff and I'm really liking what you are doing with SAT.

**Reply by sergeantd83** [reply]
Thanks for sharing this video. SAT weather compensation algorithm part works the same way. But we created our own heating curves and the parallel adjustment is proportional to the heating curve. Then SAT adds the load compensation part. This means that SAT is a Weather + Load compensator which monitors the outside and the room temperature always and carefully calculates the control Setpoint. Then, in order to apply the Low-load control algorithm,SAT monitors the boiler flow water and the return water temperatures. As a result SAT calculates the ON/OFF times in 15 minutes intervals ( SAT is also able to extend the duty cycle if the control Setpoint is too high or too low ). These two developments work tightly together in order to prevent cycling and maintain the room temperature on setpoint. Moreover we chose the 15 min duty cycle instead of 10 min that the most commercial thermostats use, this helps to reduce the ON/OFF times of the boiler by 50% during Low-load control.

**Comment by dexterionut** [comment]
Can you please also add `Immergas Victrix Omnia 25`? I'm using esp-opentherm + SAT on this boiler and it works as expected. Boiler member id is 27.

**Reply by Tikitaka23** [reply]
Hi, could I PM you for some more info about your configuration?

**Comment by bjornmorsman** [comment]
Can you please add Itho Daalderop Base Cube with memberid 29

SAT works as expected

thank you

**Reply by sergeantd83** [reply]
We are happy to hear that SAT is working properly with your boiler. Soon your boiler will be added. Thanks!

**Comment by adipierro** [comment]
Baxi ECO4s (member ID 27) and does not support max relative modulation parameter, neither it does report return water temperature 😞

**Reply by sergeantd83** [reply]
Do you have an OTGW board or OpenTherm adapter with ESPHome?

Regarding the no max modulation management we have the solution 🙂. You need to update to the latest `develop` version, enable Dynamic Minimum setpoint in advanced settings and in general settings choose the boiler temperature option. Please report back if this solution works for you.

**Reply by adipierro** [reply]
hi! My boiler is connected with ESPhome adapter. The boiler responds with 'unknown data id' on attempts to write to ID14.
Does Dynamic Minimum Setpoint work with boilers that don't have water return temperature sensor? I was under assumption that it relies on it.

```
[10:57:00.480][D][opentherm:342]: Sending request with id 14 (MAX_MODULATION_LEVEL)
[10:57:00.483][D][opentherm:564]: 00000001 00001110 00000000 00000000
[10:57:00.488][D][opentherm:566]: type: WRITE_DATA; id: 14; HB: 0; LB: 0; uint_16: 0; float: 0.000000
[10:57:00.632][D][opentherm:133]: Received OpenTherm response with id 14 (MAX_MODULATION_LEVEL)
[10:57:00.633][D][opentherm:564]: 00000111 00001110 00000000 00000000
[10:57:00.636][D][opentherm:566]: type: UNKNOWN_DATAID; id: 14; HB: 0; LB: 0; uint_16: 0; float: 0.000000
```

**Reply by adipierro** [reply]
ah! I see, there is a selector which you have mentioned! Will test and report back. Thank you :)

**Reply by adrianhomeassistant-code** [reply]
Hi, got an Ideal Logic Combi ESP1 35, I am on the latest develop version ( I think so... because although I did the HA update to develop, it still shows 4.2.0) , and I get the same (with dynamic min setpoint in advanced and boiler temp instead of return temp)

[16:08:12.630][D][opentherm:342]: Sending request with id 14 (MAX_MODULATION_LEVEL)
[16:08:12.632][D][opentherm:564]: 00000001 00001110 00000000 00000000
[16:08:12.632][D][opentherm:566]: type: WRITE_DATA; id: 14; HB: 0; LB: 0; uint_16: 0; float: 0.000000
[16:08:12.864][D][opentherm:133]: Received OpenTherm response with id 14 (MAX_MODULATION_LEVEL)
[16:08:12.867][D][opentherm:564]: 00000111 00001110 00000000 00000000
[16:08:12.869][D][opentherm:566]: type: UNKNOWN_DATAID; id: 14; HB: 0; LB: 0; uint_16: 0; float: 0.000000

**Comment by xtramotorsport** [comment]
Baxi Luna Comfort HT 1.280 with OTGW (0.10.2+50c3ed2):

<img width="829" height="45" alt="Screenshot 2026-01-08 163826" src="https://github.com/user-attachments/assets/18605757-2a74-4a0b-aaff-b153d9c86a76" />

Worked fine since September with up to Version 4.1.2, but after upgrade to V4.2.1 half sensors was gone in HA, boiler got stuck on max power, but Thermostat was not showing, that boiler is working, also OTGW was showing Relative Modulation Level 0%

and going back to V4.1.2 the following are missing too, even reconfigured SAT:
binary_sensor.boiler_health
binary_sensor.flame_health
sensor.flame_status

**Reply by Alexwijn** [reply]
Please create an issue of this and also add some logs for us to investigate. Thanks!

**Reply by xtramotorsport** [reply]
> Please create an issue of this and also add some logs for us to investigate. Thanks!

after upgrade HA to 2026.1.0, I tried again to update again to V4.2.1. So far everything works.

**Reply by Alexwijn** [reply]
Which version did you had before?

**Reply by xtramotorsport** [reply]
> Which version did you had before?

it was 2025.12.5

**Comment by Crash-Evans** [comment]
Works correctly with Ideal Logic ESP1 35KW, ID 6

**Reply by adrianhomeassistant-code** [reply]
hey @Crash-Evans , what exactly works correctly in your case? I have the exact same boiler and mine seems to be cycling quite often, every 2 min for 20 seconds, and the overall relative modulation stays around 30-40%, rarely dropping to ~26% and boiler power shows around 15kW minimum.

**Reply by Crash-Evans** [reply]
What's your current setup Thermostat/OTGW? What have you set as your OPV?



> hey @Crash-Evans , what exactly works correctly in your case? I have the exact same boiler and mine seems to be cycling quite often, every 2 min for 20 seconds, and the overall relative modulation stays around 30-40%, rarely dropping to ~26% and boiler power shows around 15kW minimum.

**Reply by adrianhomeassistant-code** [reply]
ESPhome on Diyless Opentherm shield , OPV 33.3 from my calculations

**Reply by Crash-Evans** [reply]
> ESPhome on Diyless Opentherm shield , OPV 33.3 from my calculations

@adrianhomeassistant-code , unless I am mistaken and @sergeantd83  maybe able to confirm but the DIYless Opentherm Shield ins't supported.

I have my OPV set to 50 if that helps to start you off.

**Comment by Crash-Evans** [comment]
@adrianhomeassistant-code Here are my settings if that helps?
<img width="626" height="824" alt="Screenshot 2026-01-11 at 19 18 42" src="https://github.com/user-attachments/assets/85bc5b6e-662a-40ac-b2a9-1f3b56575be0" />
<img width="616" height="813" alt="Screenshot 2026-01-11 at 19 18 19" src="https://github.com/user-attachments/assets/2f41ee06-b1ad-4c7e-8507-6b9c7f1d9b9a" />
<img width="548" height="729" alt="Screenshot 2026-01-11 at 19 17 45" src="https://github.com/user-attachments/assets/73c21719-6c85-4099-99da-fceb94bff19d" />
<img width="516" height="827" alt="Screenshot 2026-01-11 at 19 17 17" src="https://github.com/user-attachments/assets/6ade5076-daaa-489a-ab23-20c5be93a5eb" />
<img width="602" height="821" alt="Screenshot 2026-01-11 at 19 16 40" src="https://github.com/user-attachments/assets/bf094a90-558e-4126-916a-b4c7684cc2eb" />

**Reply by adrianhomeassistant-code** [reply]
Thanks @Crash-Evans , I assume you are using an OTGW MQTT ? if so may I ask which one ?

**Reply by Crash-Evans** [reply]
I am using the NodoShop V2.3
